spring-boot-2/src/main/java/org/quickperf/web/spring/QuickPerfAfterRequestServletFilter.java

```diff
@@ -199,8 +200,10 @@ private void quickPerfProcessing(HttpServletRequest httpServletRequest, HttpServ
             infoReport.append("* SQL");
             infoReport.append(lineSeparator());
             String sqlExecutionsAsString = sqlExecutions.toString();
-            String sqlExecutionsWithoutThreeLastLineBreaks = sqlExecutionsAsString.substring(0, sqlExecutionsAsString.length() - 3);
-            infoReport.append(sqlExecutionsWithoutThreeLastLineBreaks);
+            if(!StringUtils.isEmpty(sqlExecutionsAsString)) {
```

Comment:
Perhaps we could move the check before` infoReport.append("* SQL");`?

Reply:
I ve just put it in the beginning.